cloudformation/build/deployment-pipeline.yaml

```diff
@@ -518,24 +582,32 @@ Resources:
         SlackWebhook: !Ref SlackWebhook
         Project: !Ref Project
         Owner: !Ref Owner
-        Pipeline: !Ref Pipeline
+        Pipeline: !Ref BuildPipeline
```

Comment:
I think we'll have to update the notifications to pass in both pipelines?

Reply:
Oh, at first glance I thought that the notification just had to do with the source trigger from Github.  I can create another notification.

Reply:
Cool thx.  Basically we notify if the pipeline fails, and/or deploy to staging succeeded (indicating it's awaiting approval for push to prod) and deploy to prod succeeded